assets/src/dashboard/app/api/wpAdapter.js

```diff
@@ -28,11 +33,21 @@ const post = (path, options = {}) =>
     method: 'POST',
   });
 
+// `apiFetch` by default turns `DELETE` requests into `POST` requests
```

Comment:
REALLY? 🤦‍♂️

Reply:
It's actually not too bad of a thing.

It does so because many hosting providers and firewalls don't support `DELETE` requests.

Too bad some firewalls also don't support this header